distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalog.java

```diff
@@ -1189,44 +1188,6 @@ public void testUpdateContentResourceUri() throws IOException {
     deleteTemporaryFile(fileName);
   }
 
-  @Test
-  public void testCachedContentLengthHeader() throws IOException {
```

Comment:
❓ Do we have any other E2E tests for the cache? Can we run them independently now?

Reply:
I think we used to have E2E tests for the Standard UI and the cache.  I think this was just left over.  You should be able to test this independently now.